CODEOWNERS

```
@@ -0,0 +1 @@
+*	@thanujalk @brionmario @NipuniBhagya @inthirakumaaran @thivi @DimalChandrasiri @jeradrutnam @yathindrakodithuwakku
```

Check the errors here.
```suggestion
*	@brionmario @NipuniBhagya @thivi @DimalChandrasiri @jeradrutnam @yathindrakodithuwakku
```